benchmarks/test_benchmark.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import random
+import pytest
+
+benchmark = pytest.importorskip('pytest_benchmark')
+
+doubles = []
+numbers = []
+unicode_strings = []
+strings = []
+booleans = []
+list_dicts = []
+dict_lists = {}
+
+composite_object = {
+    'words': """
+        Lorem ipsum dolor sit amet, consectetur adipiscing
+        elit. Mauris adipiscing adipiscing placerat.
+        Vestibulum augue augue,
+        pellentesque quis sollicitudin id, adipiscing.
+        """,
+    'list': list(range(200)),
+    'dict': dict((str(i), 'a') for i in list(range(200))),
+    'int': 100100100,
+    'float': 100999.123456
+}
+for x in range(256):
```

Comment:
I'd prefer to have a separate block per test object.

```python
doubles = [sys.maxsize * random.random() for x in range(256)]

unicode_strings = [ "نظام  لولاية ا...رعيا لابوين عمانيين " for x in range(256)]
```

and so on.
Theadvantage would be that it's easier to reason about what the final test object might look like and adding/removing a test object just means adding/removing a block.

The disadvantage is, that there's some redundancy but I think it's worth it.
Also, we could also extract that magic number 256 to some variable.